packages/vue/src/components/list/MultiDropdownList.jsx

```diff
@@ -87,7 +89,11 @@ const MultiDropdownList = {
 
 		if (this.selectedValue) {
 			this.setValue(this.selectedValue, true);
+		} else if (this.$props.defaultValue) {
+			this.setValue(this.$props.defaultValue, true);
 		} else if (this.$props.defaultSelected) {
+			/* TODO: Remove this before next release */
+			console.warn("defaultSelected prop will be deprecated in the next release. Please replace it with defaultValue before upgrading to the next major version.");
```

Comment:
This warning message gets repeated in many places, @jyash97 can you put it out at in some util method.